src/web/app/pages-help/instructor-help-page/instructor-help-sessions-section/instructor-help-sessions-section.component.html

```diff
@@ -349,17 +349,118 @@ <h3 *ngIf="displaySubsection(showQuestion, 4, 7)">Managing Session Responses</h3
           <p>5 different views are available, each denoting the order in which responses are grouped.
             Additionally, you can group the results by team, show or hide statistics, view missing responses and filter responses from a particular section.
           </p>
-          <p>EXAMPLE BOX</p>
+          <tm-example-box>
+            <div class="card-body row">
```

Comment:
Understand that there is no standalone component that can be reused here. However, it should not remain like that for long, i.e. the standalone component should be created.

Reply:
Will create an issue to track this